apps/platform/src/components/members/membersTable/index.tsx

```diff
@@ -0,0 +1,205 @@
+'use client'
+
+import * as React from 'react'
+import { ChevronLeft, ChevronRight, ChevronsLeft, ChevronsRight } from 'lucide-react'
+import dayjs from 'dayjs'
+import { EditTwoSVG, MedalStarSVG, UserRemoveSVG } from '@public/svg/shared'
+import { useAtom, useAtomValue } from 'jotai'
+import type { GetMembersResponse } from '@keyshade/schema'
+import TransferOwnershipDialog from '../transferOwnershipDialog'
+import RemoveMemberDialog from '../removeMemberDialog'
+import EditMemberDialog from '../editMemberDialog'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow
+} from '@/components/ui/table'
+import AvatarComponent from '@/components/common/avatar'
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select'
+import { Button } from '@/components/ui/button'
+import { editMemberOpenAtom, membersOfWorkspaceAtom, removeMemberOpenAtom, selectedMemberAtom, transferOwnershipOpenAtom } from '@/store'
+
+export default function MembersTable(): React.JSX.Element {
+  const members = useAtomValue(membersOfWorkspaceAtom)
+  const [selectedMember, setSelectedMember] = useAtom(selectedMemberAtom)
+  const [isRemoveMemberOpen, setIsRemoveMemberOpen] = useAtom(removeMemberOpenAtom)
+  const [isTransferOwnershipOpen, setIsTransferOwnershipOpen] = useAtom(transferOwnershipOpenAtom)
+  const [isEditMemberOpen, setIsEditMemberOpen] = useAtom(editMemberOpenAtom)
+  const [currentPage, setCurrentPage] = React.useState(1)
+  const [rowsPerPage, setRowsPerPage] = React.useState(10)

```

Comment:
1st add type to the state, 2nd import use state from React and don't call it as method